### PR TITLE
Remove the docker-based test

### PR DESCRIPTION
**Pull request by vrabaud**
Remove the docker-based test

### CODE REVIEW

File: tests/docker/build.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ set -e
 
 # build env
 apt update
-DEBIAN_FRONTEND="noninteractive" apt install -y build-essential libjpeg-dev libpng-dev libssl-dev ninja-build cmake pkg-config git perl vim meson cargo cargo-c nasm
+DEBIAN_FRONTEND="noninteractive" apt install -y build-essential cargo cargo-c cmake git libjpeg-dev libpng-dev libssl-dev libyuv-dev meson nasm ninja-build perl pkg-config vim
```
Comment (wantehchang) [inline tests/docker/build.sh:22]: The cmake command at line 74 should be updated to say `-DAVIF_CODEC_AOM=SYSTEM`, etc. We can do that in a separate pull request.

**Review by wantehchang — APPROVED** [review]

File: tests/docker/build.sh (hunk 1 above)
Comment (vrabaud) [reply tests/docker/build.sh:22]: Indeed. BTW, what is the point of that file: shouldn't we compile the dependencies in the libavif work tree like we do now and simply install libavif? We could also install the packages for the dependencies.

Comment (wantehchang) [reply tests/docker/build.sh:22]: This tests building and installing the AV1 codecs and libavif from source code in a brand new Linux computer. We can delete this test.

**Comment by vrabaud** [comment]
Now that we have a good CI, we might as well use it. We'll revive the docker if there is ever a need for it.
